### ISSUE TITLE
Finish URDF file corresponding to actual car

### ISSUE THREAD

**Issue by chsahit** [gtagency/buzzmobile#191]
Currently, its reflecting robojacket's car's hardware

**Comment by joshuamorton** [comment]
Hasn't significant progress been made here?

**Comment by irapha** [comment]
Yes. The inertials in sahit's PR are what's missing for this to be closed I believe. It just took *forever* to get them and finalize the pr

**Comment by irapha** [comment]
The PR is ready #193, but there are a few things missing from the URDF so I don't want to close this issue right now.

Namely: #205 and #208
